app.py

```diff
@@ -272,13 +272,15 @@ def cli(
     if verbose:
         _LOGGER.setLevel(logging.DEBUG)
 
-    if not index_base_url.endswith("/"):
-        index_base_url += "/"
+    if index_base_url:
```

Comment:
It would be better to fix type hints in the function. Adding these if statements make the code less maintainable and they semantically do not correspond to what needs to be changed to conform to typing.

Reply:
It would be better to fix type hints in the function *signature*.

Reply:
solved :) thanks @fridex PTAL